spec/controllers/idv/hybrid_mobile/socure/document_capture_controller_spec.rb

```diff
@@ -259,6 +259,36 @@
         expect(response).to redirect_to(idv_hybrid_mobile_socure_document_capture_errors_url)
       end
     end
+    context 'reuse of valid capture app urls when appropriate' do
+      let(:fake_capture_app_url) { 'https://verify.socure.test/fake_capture_app' }
+      let(:socure_capture_app_url) { 'https://verify.socure.test/' }
+      let(:docv_transaction_token) { '176dnc45d-2e34-46f3-82217-6f540ae90673' }
+      let(:response_body) do
+        {
+          referenceId: '123ab45d-2e34-46f3-8d17-6f540ae90303',
+          data: {
+            eventId: 'zoYgIxEZUbXBoocYAnbb5DrT',
+            docvTransactionToken: docv_transaction_token,
+            qrCode: 'data:image/png;base64,iVBO......K5CYII=',
+            url: socure_capture_app_url,
+          },
+        }
+      end
+
+      before do
+        allow(request_class).to receive(:new).and_call_original
+        allow(I18n).to receive(:locale).and_return(expected_language)
+        allow(DocumentCaptureSession).to receive(:find_by).and_return(document_capture_session)
+      end
+
+      it 'does not create a DocumentRequest when valid capture app exists' do
+        document_capture_session.socure_docv_capture_app_url = fake_capture_app_url
+        document_capture_session.save
```

Comment:
```suggestion
        dcs = create(
          :document_capture_session,
          uuid: user.id,
          socure_docv_capture_app_url: fake_capture_app_url,
        )
        allow(DocumentCaptureSession).to receive(:find_by).and_return(dcs)
```

Reply:
this [comment](https://github.com/18F/identity-idp/pull/11555#discussion_r1866334828) should also be addressed here in the hybrid spec